Update GDExtension tutorial for entry functions (#7378)
godotengine/godot-docs: tutorials/scripting/gdextension/gdextension_cpp_example.rst

    android.release.arm64 = "res://bin/libgdexample.android.template_release.arm64.so"

This file contains a ``configuration`` section that controls the entry function of the module.

> You are also able to set a minimum compatible version with ``compatability_minimum`` of Godot
> which prevents older version of Godot to load your extension.

Reviewer: IMO we should be a little more assertive about this. Maybe:

```suggestion
You should also set the minimum compatible Godot version with ``compatability_minimum``,
which prevents older version of Godot from trying to load your extension.
```

I also snuck in a couple unrelated wording changes that sound more natural to me, but you don't have take those. :-)

My main suggestion is switching from "You are also able" to "You should" which is a little more assertive.

Author: Thanks for the feedback! I put it in because I think it does sound better than my version
